Assets/MixedRealityToolkit/Services/BaseDataProviderAccessCoreSystem.cs

```diff
@@ -164,6 +204,23 @@ public virtual IMixedRealityDataProvider GetDataProvider(string name)
             }
             catch (Exception e)
             {
+                if (retryWithRegistrar && (e is MissingMethodException))
+                {
+                    // Missing method exceptions are intentionally re-thrown so that registrar's can handle
```

Comment:
```suggestion
                    // Missing method exceptions are intentionally re-thrown so that registrars can handle
```

Reply:
this comment is wrong (from an older version of this change)... we do not actually rethrow

Reply:
removed